src/styles/blocks/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
```

Comment:
It is a good tone to set this property for all(*) elements and don't change without extra need